lib/proxy/request.rb

```diff
@@ -39,8 +39,13 @@ def create_post(path, body)
   end
 
   class ForemanRequest
+    include Proxy::Log
+
     def send_request(request)
       http.request(request)
+    rescue Exception => e
+      logger.error("Error when accessing %s: %s" % [uri.to_s, e])
```

Comment:
I'd also suggest logging the exception backtrace, unless that's going to happen further up the stack?